porcupine/pluginmanager.py

```diff
@@ -263,5 +263,3 @@ def create_dialog() -> tuple[tkinter.Toplevel, PluginDialogContent]:
     return (dialog, content)  # content returned for tests
```

Comment:
Earlier in this file there is code that asks "Do you really want to disable this plugin manager" if you try to disable it. We can delete it now, because disabling it is no longer possible.